vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/component/spreadsheet/client/js/SpreadsheetJsApi.java

```diff
@@ -480,11 +515,38 @@ public void setPopups(String raw) {
                                 .getElement()));
                 widget.setPopupWidth(state.popupWidth);
                 widget.setPopupHeight(state.popupHeight);
-                spreadsheetWidget.addPopupButton(widget);
+            } else {
+                widget = popupButtonWidgets.get(key);
             }
+            spreadsheetWidget.addPopupButton(widget);
         });
     }
 
+    public static class ContentWidget extends Widget {
+        public ContentWidget(Element contentElement) {
+            setElement(contentElement);
+        }
+    }
```

Comment:
This is needed to wrap the content element into a `Widget` instance. Not sure if there's a better approach to this.